utils/constants.tsx

```diff
@@ -763,28 +764,29 @@ export const TELEMETRY_LABELS = {
 };
 
 export const PostTypes = {
-    JOIN_LEAVE: 'system_join_leave' as const,
-    JOIN_CHANNEL: 'system_join_channel' as const,
```

Comment:
not sure why we did like this in first place

Reply:
@M-ZubairAhmed I would guess that `PostTypes.JOIN_LEAVE` would be a `const` and could not be written over